pyresttest/resttest.py

```diff
@@ -110,6 +110,7 @@ class TestConfig:
     interactive = False
     verbose = False
     ssl_insecure = False
+    non_persistent = False # Disables keep-alive pycurl option
```

Comment:
Would be nice to specify testing -vs- benchmarking here. Not sure how hard that would be though.